components/fxa-client/ios/FxAClient/FxAccountManager.swift

```diff
@@ -237,6 +237,18 @@ open class FxAccountManager {
         }
     }
 
+    /// Fetch the AET (Account Ecosystem Telemetry) user ID if available.
```

Comment:
nit: "user id" => "anonymized user id" or similar here. (I'm being pedantic here because there are a lot of identifiers in this system with different properties and it's important that we be able to keep the straight in our heads).